app/src/androidTest/java/org/mozilla/fenix/ui/CollectionTest.kt

```diff
@@ -68,6 +69,7 @@ class CollectionTest {
 
     @SmokeTest
     @Test
+    @Ignore("Failing in Move GridViewHolder to compose: https://github.com/mozilla-mobile/fenix/pull/25996")
```

Comment:
Should we open a ticket to follow up the update of the tests? :)

Reply:
Opened https://github.com/mozilla-mobile/fenix/issues/26087 for this.